### ISSUE TITLE
Is it possible to run open-datastudio Zeppelin on your own K8S cluster?

### ISSUE THREAD

**Issue by rogargon** [open-datastudio/zeppelin#2]
From the documentation, I see that Zeppelin can be run on minikube. Is it also possible to run it on your own cluster?

**Comment by Leemoonsoo** [comment]
Yes. Although each cluster environment is a little different and may require small changes sometimes, there's no reason why it can not run on any other Kubernetes cluster the same way it runs on minikube.
